packages/storybook/src/components/pricing-grid.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { fromEvent, Subscription } from 'rxjs';
 import { takeUntil, tap } from 'rxjs/operators';
-import { XGrid, ColDef, GridApi, GridOptionsProp } from '@material-ui/x-grid';
+import { XGrid, ColDef, GridOptionsProp, useApiRef } from '@material-ui/x-grid';
```

Comment:
Regarding the name of the modules, I'm pretty confident that they should be prefixed with Grid to avoid all future conflict of modules from `@material-ui/x`.

```
import { XGrid, GridColDef, GridOptionsProp, useGridApiRef } from '@material-ui/x-grid';
```